packages/mdc-layout-grid/README.md

```diff
@@ -0,0 +1,178 @@
+# mdc-layout-grid
+
+MDC layout grid is a CSS-only component that implements the
+[Material Design layout grid guidelines](https://material.io/guidelines/layout/responsive-ui.html#responsive-ui-grid),
+and makes them available to developers as CSS classes and Sass mixins.
+
+## Installation
+
+```
+npm install --save @material/layout-grid
+```
+
+## The layout grid
+
+A grid consists of a group of cells, which get positioned in sequence according to a predefined number of columns.
+Cells specify how many columns to span (the default being 4), and get placed side by side while there is room. When
+there isn't enough room for a cell, it gets moved to the beginning of the next row, and placement continues as usual.
+
+The grid has 12 columns in desktop mode, 8 columns in tablet mode, and 4 columns in phone mode. Column widths are
+variable; it's the margins and gutters that are fixed, with columns taking up the remainder of the space.
+
+Margins (the space between the edge of the grid and the edge of the first cell) and gutters (the space between edges of
+adjacent cells) are user-definable, with the Material Design spec recommending 8px, 16px, 24px or 40px as the sizes to
+choose from. The default margin and gutter are both 16px, but they don't have to be the same size.
+
+The grid and cells are not styled in any way, serving only for alignment and positioning of elements.
+
+
+## CSS class usage
+
+```html
+<div class="mdc-layout-grid">
+  <div class="mdc-layout-grid__cell"></div>
+  <div class="mdc-layout-grid__cell"></div>
+  <div class="mdc-layout-grid__cell"></div>
+</div>
+```
+
+The grid should have the `mdc-layout-grid` class, and every cell should have the `mdc-layout-grid__cell` class.
+Behavior for grids containing direct children without the `mdc-layout-grid__cell` class is undefined.
+
+### Margins and gutters
+
+The default size for both the margins and gutters in MDC layout grid is 16px.
+
+You can change the margins and gutters for a grid using the `--mdc-layout-grid-margin` and `--mdc-layout-grid-gutter`
+custom properties, respectively. This requires support for CSS custom properties on the end-user's browser.
+
+The Material Design spec recommends 8px, 16px, 24px or 40px as the sizes to choose from, but MDC layout grid doesn't impose any restrictions.
+
+```css
+.my-grid {
```

Comment:
I consider that very different usage, covered by the Sass mixins section. Unless you're talking about changing the global Sass variables we use internally, which I would consider unsupported usage.